plugins/CivilTongueEx/class.civiltongueex.plugin.php

```diff
@@ -214,8 +221,8 @@ public function discussionsController_render_before($sender) {
         $discussions = $sender->data('Discussions', []);
         if (is_array($discussions) || $discussions instanceof \Traversable) {
             foreach ($discussions as &$discussion) {
-                $discussion->Name = $this->replace($discussion->Name);
-                $discussion->Body = $this->replace($discussion->Body);
+                $discussion->Name = $this->contentFilter->replace($discussion->Name);
```

Comment:
We are calling $this->replace function that calls contentFilter all over the plugin, can't we do the same here?

Reply:
Sure.